Cargo.toml

```diff
@@ -13,6 +13,7 @@ tempfile = "3.13.0"
 serde_cbor = "0.11.2"
 lz4_flex = { version = "0.11", default-features = false }
 rand = "0.8.5"
+priority-queue = "2.1.1"
```

Comment:
FYI when merging into core, we will use our own priority implementation wrapping a std::MinHead